src/legacy/core_plugins/kibana/public/home/components/welcome.tsx

```diff
@@ -137,21 +112,34 @@ export class Welcome extends React.PureComponent<Props, State> {
           <div className="homWelcome__content homWelcome-body">
             <EuiFlexGroup gutterSize="l">
               <EuiFlexItem>
-                {shouldShowTelemetryOptIn && step === 0 && (
-                  <TelemetryOptInCard
-                    urlBasePath={urlBasePath}
-                    fetchTelemetry={fetchTelemetry}
-                    onConfirm={this.handleTelemetrySelection.bind(this, true)}
-                    onDecline={this.handleTelemetrySelection.bind(this, false)}
+                <SampleDataCard
+                  urlBasePath={urlBasePath}
+                  onConfirm={this.onSampleDataConfirm}
+                  onDecline={this.onSampleDataDecline}
+                />
+                <EuiSpacer size="s" />
+                <EuiTextColor className="euiText--small" color="subdued">
+                  <FormattedMessage
+                    id="kbn.home.dataManagementDisclaimerPrivacy"
+                    defaultMessage="To learn about how usage data helps us manage and improve our products and services, see our "
                   />
-                )}
-                {(!shouldShowTelemetryOptIn || step === 1) && (
-                  <SampleDataCard
-                    urlBasePath={urlBasePath}
-                    onConfirm={this.onSampleDataConfirm}
-                    onDecline={this.onSampleDataDecline}
+                  <EuiLink href="#">
+                    <FormattedMessage
+                      id="kbn.home.dataManagementDisclaimerPrivacyLink"
+                      defaultMessage="Privacy Policy."
+                    />
+                  </EuiLink>
+                  <FormattedMessage
+                    id="kbn.home.dataManagementDisableCollection"
+                    defaultMessage=" To disable collection, "
                   />
-                )}
+                  <EuiLink href="#">
+                    <FormattedMessage
+                      id="kbn.home.dataManagementDisableCollectionLink"
+                      defaultMessage="click here."
+                    />
```

Comment:
Clicking here just causes the page to refresh and ends up in the same place. 🤔

Reply:
Same with the privacy policy link, which seems like it should go to https://www.elastic.co/legal/telemetry-privacy-statement , but perhaps we're waiting for updated links ...

Reply:
Yup, these are just placeholders for now, thanks for calling it out!

Reply:
Added links!